graylog2-web-interface/src/components/common/ColorPickerPopover.md

```diff
@@ -1,18 +1,17 @@
 ```js
-import createReactClass from 'create-react-class';
 import { Button } from 'components/bootstrap';
 
-const ColorPickerOverlayExample = createReactClass({
-  getInitialState() {
-    return {
+class ColorPickerOverlayExample extends React.Component {
+  constructor(props) {
+    this.state = {
       color: undefined,
     };
-  },
+  }
 
-  handleColorChange(color, _, hidePopover) {
+  handleColorChange = (color, _, hidePopover) => {
```

Comment:
It looks like this syntax is not supported in our component library and we either need to use the previous syntax or adjust the styleguidist webpack config.

![image](https://github.com/user-attachments/assets/8c1f09ae-ea32-453f-af1e-0a7266f5fe66)